scripts/romanisim-make-stack

```diff
@@ -0,0 +1,302 @@
+#!/usr/bin/env python
+
+import argparse
+from copy import deepcopy
+from astropy.coordinates import SkyCoord
+from astropy import units as u
+from astropy.table import Table
+from astropy.time import Time
+import galsim
+from romanisim import wcs, persistence
+from romanisim.parameters import default_parameters_dictionary
+from romanisim import log
+from romanisim import parameters
+import ris_make_utils as ris
+
+# Radial distance from WFI_CEN to extend object generation in order to cover FOV (degrees)
+WFS_FOV = 0.6
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description='Make a stack of demo images.',
+        epilog='EXAMPLE: %(prog)s mosaic_list.csv',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+    # Command line Argument
+    parser.add_argument('input_file_name',
+                        type=str,
+                        metavar='mosaic_list.csv',
+                        help='Input (csv) file containing lists of observation parameters: '
+                             'ra, dec, roll_angle, optical_element, date, overhead_time, '
+                             'ms_table_number')
+
+    # Boresight
+    parser.add_argument('-b', '--boresight',
+                        action='store_true',
+                        help=('RA & Dec specifies location of boresight, not center of WFI.'))
+
+    # WCS Object Catalog
+    parser.add_argument('-c', '--catalog',
+                        type=str,
+                        metavar='object_table.csv',
+                        default=None,
+                        help='Object catalog file for wcs matching (csv)')
+
+    # Debug argument
+    parser.add_argument("--debug",
+                        action='store_true',
+                        help='Display verbose debug output')
+
+    # Date & Time argument
+    parser.add_argument('-d', '--date',
+                        type=str,
+                        default='2026-01-01T00:00:00',
+                        help=('UTC Date and Time of observation to simulate in ISOT format.'))
+
+    # Flux factor argument
+    parser.add_argument('-f', '--flux_factor',
+                        type=float,
+                        help='An optional flux factor for generated objects to make them brighter.')
```

Comment:
The way to do this is to control the magnitude limit in the catalog generation.  Let's not put a separate mechanism here.  I could imagine plumbing more into the catalog generation call, but I think we probably want to go the opposite direction and force people to generate a real catalog to use with their simulations rather than rely on an automatic made up one.

Reply:
Sounds good - removing.